### ISSUE TITLE
Template for blog posts

### ISSUE THREAD

**Issue by cararthompson** [rfortherestofus/marketing#66]
@dgkeyes This may be the wrong place to post this, so feel free to move it, but do you have a template Rmd you'd like us to use for blog posts?

**Comment by dgkeyes** [comment]
No, no template. The R for the Rest of Us website uses WordPress. I don't have any automated way to copy stuff. I'll probably just do it by hand.
